R/serialization.R

```diff
@@ -0,0 +1,41 @@
+# This file will be autogenerated.
+
+#' Serialize a model to the given filename.
+#'
+#' @title Serialize.
+#' @param filename Input filename.
+#' @param model_in Input model.
+Serialize <- function(filename, model_in) {
+  if (attributes(model_in)$type == "GaussianKernel") {
+    con <- file(as.character(filename), "wb")
+    serialize(SerializeGaussianKernelPtr(model_in), con)
+    close(con)
+  }
+  else if (attributes(model_in)$type == "LARS") {
+    con <- file(as.character(filename), "wb")
+    serialize(SerializeLARSPtr(model_in), con)
+    close(con)
+  }
+}
+
+
+#' Unserialize a model to the given filename.
+#'
+#' @title Unserialize.
+#' @param filename Input filename.
+#' @return model_ptr Output model.
```

Comment:
add here:

```r
#' @export
#' @rdname mlpack-serialization
```
This links the help documentation to the file created for _serialization_. You can merge the title and drop the `@param filename`.

Reply:
Done.